modules/customizationService.js

```diff
@@ -28,40 +31,46 @@ module.exports.initialize = id => {
  */
 module.exports.edit = (id, customization) => {
 	return new Promise((resolve, reject) => {
-		CustomizationModel.updateOne({ owner: id }, customization, (err, resp) => {
+		CustomizationModel.findOneAndUpdate({ owner: id }, customization, (err, resp) => {
 			if (err) {
 				reject(err);
 			} else {
-				let customSass = sass.renderSync({
-					data: `
-						@import "node_modules/bootstrap/scss/_functions";
-						
+				try {
+					let customSass = sass.renderSync({
+						data: `
+							@import "node_modules/bootstrap/scss/_functions";
+
+							
+							$useDefaultFont: ${customization.useThemeFont};
+							$font-family-sans-serif: Arial, Helvetica, sans-serif;
+
+							$useDefaultColours: ${customization.useThemeColour};
+
+							$primary: #${customization.primaryColor};
+							$secondary: #${customization.secondaryColor};
+							$light: #${customization.lightColor};
+							$dark: #${customization.darkColor};
+							
 						
-						$theme-colors: (
-							"primary": #${customization.primaryColor},
-							"secondary": #${customization.secondaryColor}
-						);
-		
-						.hover:hover {
```

Comment:
I had previously used this for styling on the products page, the hover animation will no longer appear with this removed? how would we implement the same thing using sass?